### ISSUE TITLE
fig dialouge not showing in iterm

### ISSUE THREAD

**Issue by jdockeray** [withfig/fig#2532]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.0.1 (21A559)

### Expected behaviour

When I start typing in iterm2 the fig dialogue appears and gives suggestions.

### Actual behaviour

nothing happens

### Steps to reproduce

install fig, open iterm2 with zsh and start typing.

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - chip-id: Apple M1 Max
  - cores: 10
  - mem: 64.00 GB
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - exe-path: /usr/local/bin/fig
  - install-method: brew
  - env-vars:
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.16.0
    - SHELL: /bin/zsh
    - TERM: xterm-256color
```

**Comment by mschrage** [comment]
Did you run `fig doctor`?

**Comment by jdockeray** [comment]
Yes, and it says no problems. On looking, I think it's the same issue as https://github.com/withfig/fig/issues/2522. I have run fig restart multiple times and reinstalled it, but I haven't tried upgrading the OS. I will try that and let you know if that fixes it.
